src/Dapr.Actors/Runtime/ConverterUtils.cs

```diff
@@ -68,5 +71,69 @@ public static string ConvertTimeSpanValueInDaprFormat(TimeSpan? value)
 
             return stringValue;
         }
+        
+        public static string ConvertTimeSpanValueInISO8601Format(TimeSpan value, int repetitions)
+        {
+            if (repetitions <= 0)
+            {
+                return ConvertTimeSpanValueInDaprFormat(value);
+            }
```

Comment:
If someone is calling this with 0 repetitions, would it be better to error out here as they may have expected 8601? Or is this now the default with 0 signifying the old way?

Reply:
Fixed